static/src/javascripts/projects/common/modules/article/rich-links.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+
+import type { SpacefinderRules } from 'common/modules/spacefinder';
+
+import fastdom from 'lib/fastdom-promise';
+import config from 'lib/config';
+import { isBreakpoint, getBreakpoint } from 'lib/detect';
+import fetchJson from 'lib/fetch-json';
+import mediator from 'lib/mediator';
+import reportError from 'lib/report-error';
+import template from 'lodash/utilities/template';
+import { spaceFiller } from 'common/modules/article/space-filler';
+import richLinkTagTmpl from 'raw-loader!common/views/content/richLinkTag.html';
+
+const richLinkTag = template(richLinkTagTmpl);
```

Comment:
Would you be interested in inlining the template and use an ES6 template literal instead? It's the only placed where it's used ...